src/platform_impl/windows/window_state.rs

```diff
@@ -192,13 +189,17 @@ impl WindowFlags {
 
         let (mut style, mut style_ex) = (0, 0);
 
+        style |= WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_SYSMENU | WS_CAPTION | WS_MINIMIZEBOX;
+        style_ex |= WS_EX_ACCEPTFILES;
+
         if self.contains(WindowFlags::RESIZABLE) {
             style |= WS_SIZEBOX | WS_MAXIMIZEBOX;
         }
         if self.contains(WindowFlags::DECORATIONS) {
-            style |= WS_CAPTION | WS_MINIMIZEBOX | WS_BORDER;
-            style_ex = WS_EX_WINDOWEDGE;
+            style |= WS_BORDER;
+            style_ex |= WS_EX_WINDOWEDGE;
         }
+
```

Comment:
Nit: What's the deal with this blank line?

Reply:
that was by mistake, will remove it.